lib/common/actions/index.js

```diff
@@ -148,37 +148,48 @@ function getErrorMessageFromJson (
       action = 'DEFAULT'
     }
   }
+  console.error(errorMessage)
   return { action, detail, message: errorMessage }
 }
 
+/*
+ * This component displays an error message with links to individual patterns that are
+ * used by the stop being deleted. Use of this component is a hack itself, so this should
+ * be removed when a better solution is put in place.
+ * For discussion see original PR: https://github.com/ibi-group/datatools-ui/pull/943
+ */
 const PatternLinkErrorMessage = (props) => {
   const url = window.location.href
   const basePath = url.split('edit/')[0] // Remove the current component from the URL
   const {detail, patterns} = props
-
-  // Limit the list of patterns to 5 to not overwhelm users
-  const unDisplayedPatterns = patterns.length - 5
-  const limitedPatterns = patterns.slice(0, 5)
-
   // $FlowFixMe
   return <div>
     {detail}
-    {limitedPatterns.map((pattern, index) => {
-      if (index > 5) return
-      // Pattern contains route and pattern data in format {pattern-route}
-      pattern = pattern.slice(1, -1) // Remove curly braces
-      const [patternId, routeId] = pattern.split('-')
-
-      const patternPath = `edit/route/${routeId}/trippattern/${patternId}`
-      return (
-        <li style={{listStyle: 'none'}}>
-          <a href={basePath + patternPath}>
-            {`Pattern ${patternId}`}
-          </a>
-        </li>
-      )
-    })}
-    {unDisplayedPatterns > 0 && `... and ${unDisplayedPatterns} other patterns`}
+    <ul style={{
+      background: 'aliceblue',
+      border: 'solid',
+      borderColor: 'lightgray',
+      borderRadius: '5px',
+      borderWidth: '2px',
+      maxHeight: '150px',
+      margin: '10px 0px',
+      overflow: 'auto',
+      padding: '5px 10px'
+    }}>
+      {patterns.map((pattern, index) => {
+        // Pattern contains route and pattern data in format {pattern-route}
+        pattern = pattern.slice(1, -1) // Remove curly braces
+        const [patternId, routeId] = pattern.split('-')
```

Comment:
Just to future-proof this, would it be possible to move these lines out of this component and pass it in as props instead?

Reply:
ur wish is my command https://github.com/ibi-group/datatools-ui/pull/943/commits/1203494986f3560476ac9251715238cb5c4a9fc1